docs/docusaurus/docs/cloud/connect/connect_python.md

```diff
@@ -84,6 +87,16 @@ Environment variables securely store your GX Cloud access credentials.
 
     The code example uses the default Data Source for Pandas to access the `.csv` data from the file at the specified URL path.
 
+    Alternatively, if you have already configured your data in GX Cloud you can use it instead.  To see your available Data Sources, run:
+
+    ```title="Python" name="docs/docusaurus/docs/cloud/connect/connect_python.py - list data sources"
+    ```
+  
+    Using the printed information you can get the name of one of your existing Data Sources, one of its Data Assets, and the name of a Batch Definition on the Data Asset.  Then, you can retrieve a Batch of data by updating the values for `data_source_name`, `data_asset_name`, and `batch_definition_name` in the following code and executing it:
```

Comment:
It might be beneficial here to have a sample output from the print statement so users have something to refer to, but I'll defer to your judgment.

Reply:
I'm adding a follow up ticket for that.  As it is, I don't want to bloat the current guide with too much content dedicated to a side-path from the example that is used from start to finish; I would rather we get that information added into the API guide for the list Data Sources method and then link to it there.